src/script/stores/complianceStore.ts

```diff
@@ -74,6 +75,14 @@ export function manageCookies() {
 
 export const consent = writable<CookieConsent | undefined>(undefined);
 
+export const setAsSeenRedirectDialog = () => {
+  tryLocalStorageSet('seenRedirectDilaog', 'true');
+};
+
+export const hasSeenRedirectDialog = () => {
+  return tryLocalStorageGet('seenRedirectDilaog') === 'true';
+};
```

Comment:
Not sure if this is the best place to put them, but have put them here

Reply:
Maybe just inline them at the point of use? That'll make this temporary code easier to delete in future. I think each will just be used once.

Reply:
I did inline it in [a previous commit](https://github.com/microbit-foundation/ml-trainer/pull/243/commits/4292f3c8af4581afbdd9af0cb6cad258f4e7cbb3), but then due to [the comment regarding only fetching browser info when the user has not seen the redirect dialog before](https://github.com/microbit-foundation/ml-trainer/pull/243#discussion_r1507330166) and [the comment about using persistantWritable](https://github.com/microbit-foundation/ml-trainer/pull/243#discussion_r1507315250), I moved it to a shared file `uiStore.ts` so that it can accessed by more than 1 file and is together with the other ui-related writables.